content/tutorials/spatial_wfs_services/index.Rmd

```diff
@@ -413,7 +410,23 @@ ggplot(bwk_hallerbos) +
 
 You can use `sf::st_write()` to save this layer in any format that is listed by `sf::st_drivers()`.
 
-Continuing from the same request we got earlier, we can download the data with `httr::GET` and `httr::write_disk()`.
+## Example 4: downloading data to disk
+
+Continuing from the previous example, we can download the data with `httr::GET` and `httr::write_disk()`.
+The only difference is that we at `outputFormat` to the request.
```

Comment:
Few typos seen (idea for a new branch, to be used for testing Github actions on merge):
L. 416: at --> add (typo)
L. 424: as we would _with_ any

Reply:
See PR #223